lib/src/widgets/bottom_bar_with_sheet_item.dart

```diff
@@ -11,9 +11,10 @@ import 'package:provider/provider.dart';
 const defaultDuration = Duration(milliseconds: 500);
 
 // ignore: must_be_immutable
-class BottomBarWithSheetItem extends StatelessWidget {
+class BottomBarWithSheetItem<T> extends StatelessWidget {
   final String label;
-  final IconData icon;
+  final T icon;
+  final bool weHaveIconData;
```

Comment:
This field must be private.

Reply:
And please change it name to **isIconData**

Reply:
I have addressed your concerns and added an assert() to verify that the passed object is IconData or a Widget